app/main/forms/brief_forms.py

```diff
@@ -41,13 +42,22 @@ def get_briefs(self):
         statuses = self.status.data or tuple(id for id, label in self.status.choices)
         lots = self.lot.data or tuple(id for id, label in self.lot.choices)
 
-        return self._data_api_client.find_briefs(
-            status=",".join(statuses),
-            lot=",".join(lots),
-            framework=self._framework_slug,
-            page=self.page.data,
-            human=True
-        )
+        # disable framework filtering when digital marketplace framework is live
+        if flask_featureflags.is_active('DM_FRAMEWORK'):
```

Comment:
to avoid this repetition you could do:

`kwargs = {framework: self._framework_slug} if active else {}`

and then:

`self._data_api_client.find_briefs(..... , **kwargs)`